rdkit/Chem/Descriptors.py

```diff
@@ -149,31 +172,34 @@ def NumRadicalElectrons(mol):
 
 
 def _ChargeDescriptors(mol, force=False):
+  """ Returns the charge descriptions of the molecule in a specific range: 2-value tuple
+  
+  """
   if not force and hasattr(mol, '_chargeDescriptors'):
     return mol._chargeDescriptors
-  chgs = rdPartialCharges.ComputeGasteigerCharges(mol)
+  rdPartialCharges.ComputeGasteigerCharges(mol)
   minChg = 500.
   maxChg = -500.
   for at in mol.GetAtoms():
     chg = float(at.GetProp('_GasteigerCharge'))
-    minChg = min(chg, minChg)
-    maxChg = max(chg, maxChg)
+    if chg < minChg:
+      minChg = chg
+    elif chg > maxChg:
+      maxChg = chg
   res = (minChg, maxChg)
   mol._chargeDescriptors = res
   return res
 
 
 def MaxPartialCharge(mol, force=False):
-  _, res = _ChargeDescriptors(mol, force)
-  return res
+  return _ChargeDescriptors(mol, force)[1]
```

Comment:
I'm not sure if this is clearer. A namedtuple would be beneficial.
```
from collections import namedtuple

ChargeDescriptors = namedtuple('ChargeDescriptors', 'minCharge, maxCharge')

def _ChargeDescriptors(mol, force=False):
  """ Returns the charge descriptions of the molecule in a specific range: 2-value tuple
  
  """
  if not force and hasattr(mol, '_chargeDescriptors'):
    return mol._chargeDescriptors
  rdPartialCharges.ComputeGasteigerCharges(mol)
  minChg = 500.
  maxChg = -500.
  for at in mol.GetAtoms():
    chg = float(at.GetProp('_GasteigerCharge'))
    if chg < minChg:
      minChg = chg
    elif chg > maxChg:
      maxChg = chg
  res = ChargeDescriptors(minChg, maxChg)
  mol._chargeDescriptors = res
  return res

def MaxPartialCharge(mol, force=False):
  return _ChargeDescriptors(mol, force).minCharge
```

Reply:
[FYI] Yes, I agree with this change, but if (just if) making this change in other module will crash it (similar to the commit made on `GraphDescriptors.py`

Reply:
I am afraid that this could break the compatibility, I would prefer this `ChargeDescriptors = namedtuple('ChargeDescriptors', ['minCharge', 'maxCharge'])`. This allowed that `ChargeDescriptors[0] = ChargeDescriptors.minCharge` and similar for `maxCharge`